src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -489,15 +489,17 @@ def _filter_privacy_request_queryset(
 
 
 def _sort_privacy_request_queryset(
-    query: Query, sort_due_date: Optional[ColumnSort]
+    query: Query, sort_field: str, sort_direction: ColumnSort
```

Comment:
I _think_ `sort_direction` might need to be `str` for this to work? Does `ColumnSort` expect a function?

Reply:
It looks like it's working without using `.value`. That might be because I add the `str` type as a mixin
```python
class ColumnSort(str, Enum):
    DESC = "desc"
    ASC = "asc"
```

I'm okay with switching it to `sort_direction.value` to help it be more clear. It's a little confusing when python enums require using the `.value` in some cases and not others. I'll update it now

Reply:
Ah great, that explains it, I didn't notice that

Reply:
It looks like that may not work. The return type of the enum value property is `Any`. The type checker throws this error for `getattr`: `Expected type 'str', got '() -> Any`.

I was able to make the type checker happy by overloading the `value` property with one that has the correct type hints
```python
class ColumnSort(str, Enum):
    DESC = "desc"
    ASC = "asc"

    @property
    def value(self) -> str:
        return self._value_
```
This _should_ be okay but I'm not 100% sure if it's worth overloading the property. What do you think @seanpreston?


Also, I double checked and the `str` mixin is what makes it work without the `.value`. Here is the error that is displayed when the mixin is removed: `Expected type 'str', got 'ColumnSort' instead `. The mixin must automatically convert it to a `str`